docs/source/dataset.rst

```diff
@@ -0,0 +1,50 @@
+Dataset
+========
+
+
+Here, we have two simulated 4D-STEM datasets containing diffraction shifting, elliptical distortion, and an r-space/k-space rotational offset.  Two HDF5 files each include the simulated data for two different electron probes: a standard probe, using a circular probe-forming aperture, and a 'bullseye' probe, using a patterned aperture.  Each HDF5 file contains the following data objects:
+
+(a) the 'experimental' 4D-STEM scan of a strained single-crystal gold nanoparticle 
+
+(b) a 4D-STEM scan of a calibration sample of polycrystalline gold 
+
+(c) a stack of diffraction images of the electron probe over vacuum 
+
+(d) a single image of the electron probe over the sample and far from focus, such that the CBED forms a shadow image 
+
+Download
+~~~~~~~~~
+
+
+1.  `bullseye probe <https://zenodo.org/record/3592520/files/calibrationData_bullseyeProbe.h5?download=1>`_
+2.  `standard probe <https://zenodo.org/record/3592520/files/calibrationData_circularProbe.h5?download=1>`_
+
+
+
+
+
+
+
+
+Creating your own dataset
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can generate your own dataset in the following way
```

Comment:
This should note that this will just generate random data, not simulate data with any meaning, so it should only be used for the simplest testing purposes.